README.md

```diff
@@ -11,49 +11,171 @@
 
 
 
-Library to propose code changes
+## Description
+Code-suggester automates the steps involved in making code changes or code suggestions to your [GitHub](https://github.com/) repository! Code-suggester
+1. can be imported as a [library](#Core-Library), or
+2. used as a [CLI](#CLI) tool
 
 
+## Core Library
 
+### Installation
 
-* [github.com/googleapis/code-suggester](https://github.com/googleapis/code-suggester)
+```bash
+npm i code-suggester
+```
+
+### Example
+
+```
+import { makePr } from "code-suggester";
+
+async function main() {
+  const octokit = new Octokit({ auth: process.env.ACCESS_TOKEN });
```

Comment:
Until this point, I had no idea this was purely focused on Probot (is it?) If it is, probably want to call that out in the top level description above ☝️

Reply:
It's not just focused on Probot. I just designed it so that it would also fit well with Probot applications. Is there anything in particular that makes it appear that way?

Reply:
Oh perfect then!